src/components/Filters/Brush.js

```diff
@@ -33,16 +29,16 @@ class Brush extends Component {
     this.brush = brushX()
         .extent([
           [margin.left, margin.top],
-          [this.state.width - margin.right, this.state.height - margin.bottom]
+          [this.props.width - margin.right, this.props.height - margin.bottom]
         ])
         .on('start', this.props.onBrushStart())
         .on('end', this.brushEnded)
         .on('brush', this.brushed)
   }
 
   componentDidMount() {
-    // console.log('componentDidMount')
-    this.setupBrush(this.state.lineGenerator, this.state.series, this.state.dates, this.state.width, this.state.height);
+    console.log('componentDidMount')
```

Comment:
I find it helpful to log where this is coming from

`console.log('Brush: componentDidMount')`